src/tests/performance/ping_pong.rs

```diff
@@ -0,0 +1,196 @@
+    // ├─────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼────────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼──────────────┤
+    // │      20 │        150 │          0 │         10 │              2 │          0 │          0 │          0 │          0 │         10 │          19.17 │      13.95 │        41.22 │
+    // ├─────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼────────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼──────────────┤
+    // │      30 │        150 │          0 │         50 │              7 │          0 │          0 │          0 │          0 │         45 │          14.60 │      15.83 │        41.52 │
+    // ├─────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼────────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼──────────────┤
+    // │      50 │        150 │          0 │         45 │              3 │          0 │          0 │          0 │          0 │          0 │          10.13 │      19.42 │        39.14 │
+    // ├─────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼────────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼──────────────┤
+    // │      75 │        150 │          0 │         57 │              6 │          0 │          0 │          0 │          0 │         41 │           8.23 │      25.28 │        36.63 │
+    // ├─────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼────────────┼────────────┼────────────┼────────────┼────────────────┼────────────┼──────────────┤
+    // │     100 │        150 │          0 │         16 │              2 │          0 │          0 │          0 │          0 │          1 │           7.21 │      29.92 │        36.13 │
+    // └─────────┴────────────┴────────────┴────────────┴────────────────┴────────────┴────────────┴────────────┴────────────┴────────────┴────────────────┴────────────┴──────────────┘
+    //
+    // *NOTE* run with `cargo test --release tests::performance::ping_pong -- --nocapture`
+
+    let synth_counts = vec![1, 10, 15, 20, 30, 50, 75, 100];
+
+    let mut table = RequestsTable::default();
+
+    let target = TempDir::new().expect("Unable to create TempDir");
+    let mut node = Node::builder()
+        .start(target.path(), NodeType::Stateless)
+        .await
+        .unwrap();
+    let node_addr = node.addr();
+
+    for synth_count in synth_counts {
+        // setup metrics recorder
+        let test_metrics = TestMetrics::default();
+        // clear metrics and register metrics
+        metrics::register_histogram!(METRIC_LATENCY);
+
+        let mut synth_handles = Vec::with_capacity(synth_count);
+        let test_start = tokio::time::Instant::now();
```

Comment:
Another option is the [`time`](https://time-rs.github.io/api/time/index.html) crate, if you'd like a more complete time-related lib; though this is fine too.